test/Li.jl

```diff
@@ -127,5 +135,5 @@ end
                 @test typeof(PolyLog.reli(n, convert(T, x))) == T
             end
         end
-    end
+    end    
```

Comment:
Sorry to be pedantic, but would you mind removing this trailing whitespace? :)

Reply:
no problem!

Reply:
Thanks! :)